integrations/aws/main.py

```diff
@@ -0,0 +1,168 @@
+    for batch in describe_resources(sessions, 'cloudformation', 'list_stacks', 'StackSummaries', 'NextToken'):
+        yield batch
+
+
+async def resync_cloudcontrol(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    sessions = _get_sessions()
+    next_token = None
+    for session in sessions:
+        region = session.region_name
+        account_id = find_account_id_by_session(session)
+        while True:
+            all_instances = []
+            try:
+                cloudcontrol = session.client('cloudcontrol')
+                params = {
+                    'TypeName': kind,
+                }
+                if next_token:
+                    params['NextToken'] = next_token
+                
+                response = cloudcontrol.list_resources(**params)
+                next_token = response.get('NextToken')
+                for instance in response.get('ResourceDescriptions', []):
+                    described = describe_single_resource(kind, instance.get('Identifier'), account_id, region)
+                    described.update({'Kind': kind, ACCOUNT_ID_PROPERTY: account_id})
+                    all_instances.append(_fix_unserializable_date_properties(described))
+                yield all_instances
+            except Exception as e:
+                logger.error(f"Failed to list CloudControl Instance in account {account_id} kind {kind} region: {region}; error {e}")
+                break
+            if not next_token:
+                break
+                
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.EC2)
+async def resync_ec2(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    sessions = _get_sessions()
+    for session in sessions:
+        region = session.region_name
+        account_id = find_account_id_by_session(session)
+        try:
+            ec2 = session.resource('ec2')
+            ec2_client = session.client('ec2')
+            for page in ec2.instances.pages():
+                page_instances = []
+                for instance in page:
+                    described_instance = ec2_client.describe_instances(InstanceIds=[instance.id])
+                    instance_definition = described_instance["Reservations"][0]["Instances"][0]
+                    instance_definition.update({ACCOUNT_ID_PROPERTY: account_id})
+                    page_instances.append(_fix_unserializable_date_properties(instance_definition))
+                yield page_instances
+        except Exception as e:
+            logger.error(f"Failed to list EC2 Instance in region: {region}; error {e}")
+            break
+
+
+@ocean.router.post("/webhook")
+async def webhook(request: Request) -> dict[str, Any]:
+    logger.info("Received webhook")
+    validate_request(request)
+    try:
+        body = await request.json()
+        logger.info("Webhook body", body=body)
+        resource_type = body.get("resource_type")
+        identifier = body.get("identifier")
+        account_id = body.get("accountId")
```

Comment:
note that by default if user is running with LOG_LEVEL=INFO he won't be able to see the extras params that you attach to the log. 
Also logging the body is a bit too much in my opinion in a `info` level.
I would prefer having a log with a context about the `resource_type`, `identifier`, `account_id`, `action` etc...